test/util/nightwatch-commands/axeCheck.js

```diff
@@ -19,16 +19,16 @@ export function command(context, config, _callback) {
   }, [axeSource]);
 
   // Run axe checks and report
-  this.executeAsync((innerContext, done) => {
+  this.executeAsync((innerContext, rules, done) => {
```

Comment:
No, though I did try  that in an earlier version: added a `rules` field to the `config` paramater in the `axeCheck` command. I think that was a bit messier because you'd have to go into your tests and update the `axeCheck` calls, whereas with this command-line, you can just see what failures are there without even editing the test.

We could actually put both capabilities, if there are e2e tests with many failures and it would be helpful to only check one page at a time for example. From what I've seen, most e2e tests only have a few failures and they're pretty easy to track down from the details provided in the failure message.